node/src/Pos/Block/Network/Logic.hs

```diff
@@ -338,6 +342,28 @@ handleRequestedHeaders cont inRecovery headers = do
             logDebug msg
             throwM $ DialogUnexpected msg
   where
+    newestHeader = headers ^. _NewestFirst . _neHead
+    oldestHeader = headers ^. _NewestFirst . _neLast
+    newestHash = headerHash newestHeader
+    oldestHash = headerHash oldestHeader
+    oldestEpoch = oldestHeader ^. epochIndexL
+
+    tryCalculateLrc = do
+        tipEpoch <- view epochIndexL <$> DB.getTipHeader @ssc
+        if oldestEpoch == tipEpoch + 1 then do
```

Comment:
@volhovM it seems that my comment is still correct: https://github.com/input-output-hk/cardano-sl/pull/1540#discussion_r139650150

If tip is before crucial slot, LRC will fail and we will log a warning, but the situation is quite normal.
Why not check that tip's slot is after crucial slot? And add reference to CSL-1660.